crates/signer/src/lib.rs

```diff
@@ -14,7 +14,7 @@ pub use signer::{Signer, SignerSync};
 
 pub mod utils;
 
-pub use alloy_primitives::Signature;
+pub use alloy_primitives::PrimitiveSignature;
```

Comment:
```suggestion
pub use alloy_primitives::PrimitiveSignature as Signature;
```

`PrimitiveSignature` is a temporary name. we wanted to avoid breaking changes in core so didn't replace the sig type right away, but this will happen on the next minor release